src/st/entity_unkId14_spawner.h

```diff
@@ -0,0 +1,24 @@
+extern u16 eUnk14SpawnRots[];
+
+void EntityUnkId14Spawner(
+    Entity* self, u8 count, u8 params, s32 x, s32 y, u8 arg5, s16 xGap) {
+    Entity* newEntity;
+    s32 i;
+    s16 newX = self->posX.i.hi + x;
+    s16 newY = self->posY.i.hi + y;
+
+    for (i = 0; i < count; i++) {
+        newEntity = AllocEntity(&g_Entities[160], &g_Entities[192]);
+        if (newEntity != NULL) {
+            newEntity->posX.i.hi = newX + xGap * i;
+            newEntity->posY.i.hi = newY;
+            newEntity->entityId = E_ID_14;
+            newEntity->pfnUpdate = EntityUnkId14;
+            newEntity->params = params;
+            newEntity->ext.generic.unk94 = arg5 + i;
+            newEntity->rotY = newEntity->rotX = eUnk14SpawnRots[arg5 + i];
```

Comment:
Can you prepend the `g_` to `eUnk14SpawnRots`? Unless it's marked as `static` and it is self-contained in a C file, it is better to be explicit the variable being globally accessible

Reply:
Ah yes of course, fixed